src/TypeSpec.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -161,7 +166,8 @@ export async function $onEmit(context: EmitContext<NetEmitterOptions>) {
                     options["generateTestProject"] === true
                         ? undefined
                         : options["generateTestProject"],
-                "use-model-reader-writer": options["use-model-reader-writer"]
+                "use-model-reader-writer": options["use-model-reader-writer"],
+                "azure-arm": sdkContext.arm
```

Comment:
can it have a default value so we don't to upgrade all configuration.json.

Reply:
updated